internal/store/postgres/alerts.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/goto/siren/pkg/errors"
 	"github.com/goto/siren/pkg/pgc"
 	"github.com/lib/pq"
+	"go.nhat.io/otelsql"
```

Comment:
we should use goto package instead of this

Reply:
Modified this is `go.mod` using go replace, will make changes for this.

Reply:
I tried this, but the otelsql package itself uses `go.nhat.io/otelsql` internally to import its packages. So we have to update every import, changing a lot of files.

```go
package otelsql

import (
	"context"
	"time"

	"go.opentelemetry.io/otel/attribute"
	"go.opentelemetry.io/otel/trace"

	xattr "go.nhat.io/otelsql/attribute" // needs to change everywhere
)
```
IMO using `replcae directive` is looking like the  better way of doing this we can sync the update from `nhatthm/otelsql` package in future without much conflicts.

Reply:
we need to update the forked too